src/app/organization/offices/create-offices/create-offices.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container">
```

Comment:
Make sure the container width is small as the form is short as per the task description. (See Accounting -> Closing Entries -> Create as an example)

Reply:
is 50% width good for the form?